neo/SmartContract/Native/Tokens/GasToken.cs

```diff
@@ -21,45 +21,8 @@ public sealed class GasToken : Nep5Token<Nep5AccountState>
 
         private const byte Prefix_SystemFeeAmount = 15;
 
-        internal GasToken() : base()
+        internal GasToken()
```

Comment:
We should call the base constructor
```suggestion
        internal GasToken() : base()
```